pf/internal/schemashim/attr_schema.go

```diff
@@ -55,7 +55,9 @@ func (*attrSchema) DefaultFunc() shim.SchemaDefaultFunc {
 	panic("DefaultFunc() should not be called during schema generation")
 }
 func (*attrSchema) DefaultValue() (interface{}, error) {
-	panic("DefaultValue() should not be called during schema generation")
+	// DefaultValue() should not be called by tfgen, but it currently may be called by ExtractInputsFromOutputs, so
+	// returning nil is better than a panic.
```

Comment:
It's not OK but I don't think we can do anything about it, because we can't detect what's a default value, see #732 

Perhaps there's some workarounds we can think of to detect?

One specific downside of this is that import spits out all the extraneous fields like 'lower=True' that would have better been omitted:

```
new_password = random.RandomPassword("newPassword",
    length=11,
    lower=True,
    number=True,
    numeric=True,
    special=True,
    upper=True,
    opts=pulumi.ResourceOptions(protect=True))
```